.github/workflows/check-pr-engine-editor-debug-and-tests.yaml

```diff
@@ -3,7 +3,7 @@ on: [pull_request]
 
 env:
   SCONS_CACHE_MSVC_CONFIG: true
-  GODOT_BASE_VERSION: 3.4.2
+  GODOT_BASE_VERSION: 3.5.0
```

Comment:
NIT: Why not replace all with `3.x` for easier search and replace?

Reply:
I think we should use tag only here, but I don't have a strong opinion about this

Reply:
I don't have either.
Feel free to merge or change it.